### ISSUE TITLE
Menu button in header does not include label in name

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1937]
The aria-label for the 'Menu' button in the header does not include 'menu' despite the button being labelled with the text 'menu'.

https://github.com/alphagov/govuk-frontend/blob/4dca6731980b031262c535f456aad3dc53a46f6e/src/govuk/components/header/template.njk#L63

This fails WCAG 2.5.3 - Label in name, as flagged in an audit of Digital Marketplace.

### Actual Result

On zooming the page to 200% and beyond, the top level navigation links displayed in the page gets collapsed into a Menu button.

When screen focus reaches the Menu button, it is declared by the screen reader as: Show or Hide Top Level Navigation button

### Expected Result

When a screen element is declared by screen reader, if the element is labelled with text, the text should be included in its accessible name.

Thanks @domoscargin for flagging this.

**Comment by 36degrees** [comment]
Suggested fix – we could change the default label to 'Show or hide navigation **menu**' or 'Toggle navigation **menu**'?

(It'd be nice to set the name dynamically to 'Show navigation menu' or 'Hide navigation menu', depending on the state of the menu, but that seems out of scope for now)
